### DISCUSSION TITLE
2009/05/05/%E9%A2%84%E8%AE%A1%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%87%BA%E7%89%88%E7%9A%84%E6%97%B6%E9%97%B4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#256] [Announcements]
# 2009/05/05/%E9%A2%84%E8%AE%A1%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%87%BA%E7%89%88%E7%9A%84%E6%97%B6%E9%97%B4

昨晚收到编辑的回复。他们收到稿件之后先要编辑加工，需要一个多月时间；然后排版，需要近两个月；然后校对，印刷，出版。等这本书上架，最快也要八月底了。

https://blog.qizhen.xyz/2009/05/05/%E9%A2%84%E8%AE%A1%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%87%BA%E7%89%88%E7%9A%84%E6%97%B6%E9%97%B4

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Fancy
2009年05月5日 11:50 编辑
你好，有事相求，能加我吗？wxy23.love@hotmail.com

回复
